app/components/dashboard/diabetes/recent_measurements_component.rb

```diff
@@ -0,0 +1,31 @@
+class Dashboard::Diabetes::RecentMeasurementsComponent < ApplicationComponent
+  include Reports::RegionsUrlHelper
+
+  attr_reader :region, :user, :recent_blood_sugars, :display_model, :page
+
+  def initialize(recent_blood_sugars:, display_model:, page:, region: nil, user: nil)
+    @region = region
+    @user = user
+    @recent_blood_sugars = recent_blood_sugars
+    @display_model = display_model
+    @page = page
+    @recent_blood_sugars.first.risk_state
```

Comment:
This failed with an `undefined method 'risk_state' for nil:NilClass` if there's no blood sugars recorded at a facility.

Reply:
This is some code we were using to debug, which has sneaked in to the the PR. Removing this.